recipe/meta.yaml

```diff
@@ -92,7 +92,7 @@ outputs:
         - eigen {{ eigen }}.*
         - eups
         - firefly-client
-        - galsim <2.4.0
+        - galsim
```

Comment:
You want a lower bound here since old galsim now won't work with the new Rubin code.

Reply:
Ah ok. If that's true then ignore me.

Reply:
Yes, we do support both.

Reply:
https://github.com/lsst/meas_extensions_shapeHSM/pull/39/files